docs/FEATURES.md

```diff
@@ -1,32 +1,75 @@
+| Accessibility             | Stops Wheelchair Accessibility   | One **wheelchair_boarding** value in [stops.txt](https://gtfs.org/documentation/schedule/reference/#stopstxt) |  https://gtfs.org/getting_started/features/accessibility/#stops-wheelchair-accessibility |
+| Accessibility             | Trips Wheelchair Accessibility   | One **wheelchair_accessible** value in [trips.txt](https://gtfs.org/documentation/schedule/reference/#tripstxt) |   https://gtfs.org/getting_started/features/accessibility/#trips-wheelchair-accessibility |
+| Base add-ons            | Route Colors              | One **color** value <br>OR<br>one text_color value in [routes.txt](https://gtfs.org/schedule/reference/#routestxt)    |    https://gtfs.org/getting_started/features/base_add-ons/#route-colors |
+| Base add-ons            | Bike Allowed              | One **bikes_allowed** value in [trips.txt](https://gtfs.org/schedule/reference/#tripstxt)  |  https://gtfs.org/getting_started/features/base_add-ons/#bike-allowed|
+| Base add-ons           | Translations              | One line of data in [translations.txt](https://gtfs.org/schedule/reference/#translationstxt)  |  https://gtfs.org/getting_started/features/base_add-ons/#translations |
+| Base add-ons             | Headsigns                 | One **trip_headsign** in [trips.txt](https://gtfs.org/schedule/reference/#tripstxt)<br>OR<br>one stop_headsign value in [stop_times.txt](https://gtfs.org/schedule/reference/#stop_timestxt)  | https://gtfs.org/getting_started/features/base_add-ons/#headsigns      |       
+| Base add-ons                | Location types            | One **location_type** value in [stops.txt](https://gtfs.org/schedule/reference/#stopstxt)    | https://gtfs.org/getting_started/features/base_add-ons/#location-types |
+| Base add-ons                | Feed Information          | One line of data in [feed_info.txt](https://gtfs.org/schedule/reference/#feed_infotxt)  |  https://gtfs.org/getting_started/features/base_add-ons/#feed-information |
+| Base add-ons                | Attributions              | One line of data in [attributions.txt](https://gtfs.org/schedule/reference/#attributionstxt) |   https://gtfs.org/getting_started/features/base_add-ons/#attributions|
+| Base add-ons                  | Shapes                    | One line of data in [shapes.txt](https://gtfs.org/schedule/reference/#shapestxt)      |   https://gtfs.org/getting_started/features/base_add-ons/#shapes |   
+| Base add-ons               | Transfers                 | One line of data in [transfers.txt](https://gtfs.org/schedule/reference/#transferstxt)     | https://gtfs.org/getting_started/features/base_add-ons/#transfers  |
+| Base add-ons | Frequency-Based Service | One line of data in [frequencies.txt](https://gtfs.org/schedule/reference/#frequenciestxt)    | https://gtfs.org/getting_started/features/base_add-ons/#frequency-based-service  |
```

Comment:
Name should be `Frequencies` (the correct name is already being used in the validator). I also realize we're being inconsistent in the documentation when referring to this feature, I'll update GTFS.org to use Frequencies across the board.

Reply:
We just changed this so we'll change it back!